lib/syncthing/internals.go

```diff
@@ -23,6 +25,17 @@ type Internals struct {
 	model model.Model
 }
 
+// Exposed version of db.Counts, which is now internal
+type Counts struct {
```

Comment:
If you make it actually identical to the source struct you can just type convert it and avoid the manual copy in `newCounts`

Reply:
Sure! (I assume in the future more stuff will be moved to `internals/`? Then we can be even more precise in which types are exported here)